scenarios/cpp/windows/captioning/caption.cpp

```diff
@@ -0,0 +1,580 @@
+/* NOTES:
+- In Windows, place the following files from the Speech SDK in the same folder as your compiled .exe:
+	- Microsoft.CognitiveServices.Speech.core.dll
+	- Microsoft.CognitiveServices.Speech.extension.audio.sys.dll
+*/
+
+/*
+Dependencies
+*/
+
+// exception
+#include <exception>
+// filesystem
+#include <filesystem>
+// ofstream
+#include <fstream>
+// fixed, setfill, setprecision, setw
+#include <iomanip>
+// cout
+#include <iostream>
+// optional
+#include <optional>
+// istringstream, ostringstream
+#include <sstream>
+
+/* Notes:
+- Install the NuGet package Microsoft.CognitiveServices.Speech.
+- The Speech SDK on Windows requires the Microsoft Visual C++ Redistributable for Visual Studio 2019 on the system. See:
+https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-sdk?tabs=windows%2Cubuntu%2Cios-xcode%2Cmac-xcode%2Candroid-studio
+*/
+#include <speechapi_cxx.h>
+
+using namespace Microsoft::CognitiveServices::Speech;
+using namespace Microsoft::CognitiveServices::Speech::Speaker;
+
+/*
+Global constants
+*/
+
+// Note: Continuous language identification is supported only with v2 endpoints.
+const std::string v2EndpointStart ("wss://");
+const std::string v2EndpointEnd (".stt.speech.microsoft.com/speech/universal/v2");
```

Comment:
Usually, we define const in the file scope when they are used in multiple places (functions). Here you only use them in one function V2EndpointFromRegion(). I think you can move these definitions into that function. 

Same for other consts if they are only used in one place.